doc/source/develop/contribute.rst

```diff
@@ -54,7 +54,6 @@ Running tests
 
 To run tests of pulp you need to run::
 
-    cd pulp
     python3 pulp/tests/run_tests.py
```

Comment:
this does not produce any error on my side.

Reply:
The file we want to run is `PROJECT_ROOT/pulp/tests/run_tests.py`, right? so if we cd into `PROJECT_ROOT/pulp` then we'd be looking for `PROJECT_ROOT/pulp/pulp/tests/run_tests.py`, which doesn't exist.

(unless you're referring to the error in the PR description, in which case I'm guessing the error I'm getting might be due to using Windows, which doesn't have linux symlinks, resulting in editable mode behaving differently? Nonetheless, errors in tests probably shouldn't block this PR IMO)